jujugui/static/gui/src/app/utils/environment-change-set.js

```diff
@@ -178,17 +178,19 @@ YUI.add('environment-change-set', function(Y) {
           //  under. In most cases this will be `env`.
           result = callback.apply(this, arguments);
         }
-        self.fire('taskComplete', {
-          id: record.id,
-          record: record,
-          result: arguments
-        });
+        document.dispatchEvent(new CustomEvent('ecs.taskComplete', {
+          detail: {
```

Comment:
`CustomEvent.detail` is how you pass data around when dispatching events (see https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent)